Flappy Bird Clone/Assets/CameraMovementTest.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraMovementTest : MonoBehaviour
+{
+    // Update is called once per frame
+    void Update()
+    {
+        transform.position += new Vector3(1, 0, 0) * (2 * Time.deltaTime);
```

Comment:
You can use `Vector3.left` as well: `new Vector3.left * (2 * Time.deltaTime);`

Reply:
Alright, gonna change it for the character movement later, thanks!